.gitreview

```diff
@@ -0,0 +1,6 @@
+[gerrit]
```

Comment:
Have removed the config by backing it out as didn't want to change the history of the base branch. Thoughts?